stereo_calibration:
  ros__parameters:

    minimum_valid_images: 10

    chessboard_size: [5, 7]
    square_size: 22.0 # mm

    image_size: [808, 608] # (width, height)
    
    images_path: /home/antonino/Desktop/work/stereo_ws/src/ROS2-Stereo-Reconstruction/calibration_images/
    calibration_path: /home/antonino/Desktop/work/stereo_ws/src/ROS2-Stereo-Reconstruction/calibration/
